operator/notifications/slack/slack.go

```diff
@@ -0,0 +1,26 @@
+package slack
```

Comment:
We've named a bunch of ship-it pkgs to collide with the "official" pkgs. It's ended up causing awkward imports whenever we're forced to import both in the same file.

I'd suggest organizing this code like `notifications/slack.go` and then have 

```
package notifications

type Slack struct { ... }
```

Which results in an obviously meaningful imported type `notifications.Slack`, whereas I don't think `slack.Manager` is as meaningful.